benchmarks/multiscale_normal.cpp

```diff
@@ -0,0 +1,29 @@
+#include "testsetup.hpp"
+
+#include <py4dgeo/compute.hpp>
+#include <py4dgeo/epoch.hpp>
+
+#include <benchmark/benchmark.h>
+
+using namespace py4dgeo;
+
+static void
+multiscale_normal_benchmark(benchmark::State& state)
+{
+  auto [cloud, corepoints] = ahk_benchcloud();
+  Epoch epoch(*cloud);
+  epoch.kdtree.build_tree(10);
+  std::vector<double> normal_radii{ 1.0 };
+  EigenNormalSet directions(corepoints->rows(), 3);
+  EigenNormalSet orientation(1, 3);
+  orientation << 0, 0, 1;
+
+  for (auto _ : state) {
+    // Precompute the multiscale directions
+    compute_multiscale_directions(
+      epoch, *corepoints, normal_radii, orientation, directions);
+  }
+}
+
+BENCHMARK(multiscale_normal_benchmark)->Unit(benchmark::kMillisecond);
```

Comment:
Given the runtime of the benchmark, the next lower display unit would be a better fit (`benchmark::kMicrosecond`).